tox.ini

```
@@ -11,7 +11,7 @@ commands =
     black src tests setup.py --check --diff --exclude "tests/fixtures/*"
     flake8 src tests setup.py
     mypy src tests setup.py --exclude "tests/fixtures/*" --namespace-packages
-    pytest --cov=ethereum --cov-report=term --cov-report "xml:{toxworkdir}/coverage.xml" --ignore-glob='tests/fixtures/*' -n 2
+    pytest --cov=ethereum --cov-report=term --cov-report "xml:{toxworkdir}/coverage.xml" --ignore-glob='tests/fixtures/*' -n 2 --basetemp=specs-temp
```

Would it make sense to put the directory under [`{temp_dir}`](https://tox.wiki/en/3.24.5/config.html#globally-available-substitutions)?

```suggestion
    pytest --cov=ethereum --cov-report=term --cov-report "xml:{toxworkdir}/coverage.xml" --ignore-glob='tests/fixtures/*' -n 2 --basetemp="{temp_dir}/pytest"
```

---

@SamWilsn Sure. As long as we don't leave it up to `pytest` to decide the temp folder, we could place it wherever we want. Have updated the command and reverted my changes to `.gitignore` since `.tox` is already in it.